control-plane/rest/src/versions/v0.rs

```diff
@@ -836,6 +844,18 @@ impl RestError {
                 );
                 HttpResponse::Unauthorized().json(error)
             }
+            ReplyErrorKind::Conflict => {
+                let error =
+                    RestJsonError::new(RestJsonErrorKind::Conflict, &details);
+                HttpResponse::Conflict().json(error)
+            }
+            ReplyErrorKind::FailedPersist => {
+                let error = RestJsonError::new(
+                    RestJsonErrorKind::FailedPrecondition,
```

Comment:
Shouldn't this be `RestJsonErrorKind::FailedPersist`?

Reply:
Good catch!